packages/web-react/src/components/Button/stories/ButtonSize.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Button from '../Button';
+
+// @see: https://github.com/storybookjs/storybook/issues/8104#issuecomment-932310244
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const Story = (props: unknown) => (
+  <>
+    <Button color="primary" size="medium">
```

Comment:
Is it necessary to specify `size` when `medium` is the default value?

Reply:
We could ask ourselves the same question with the color. 🤔 Probably not, good point. 👍🏻

Reply:
Well, we can use default values and not set those props. But I do not know if we have an agreement on what the component should look like in a default state. I think we are forcing the developers to choose first what kind of button they want to use.

Reply:
I'll mark this thread as resolved conversation and we can discuss about this topic later.